modules/dynamic-plugins/rhdh-supported-plugins.csv

```diff
@@ -1,48 +1,49 @@
 "Argo CD ","@roadiehq/scaffolder-backend-argocd","Backend","1.1.27","Community Support","./dynamic-plugins/dist/roadiehq-scaffolder-backend-argocd-dynamic","`ARGOCD_USERNAME`;`ARGOCD_PASSWORD`;`ARGOCD_INSTANCE1_URL`;`ARGOCD_AUTH_TOKEN`;`ARGOCD_INSTANCE2_URL`;`ARGOCD_AUTH_TOKEN2`;","Disabled"
+"Argo CD (Red Hat) ","@backstage/community-plugin-redhat-argocd","Frontend","1.8.0","Community Support","./dynamic-plugins/dist/backstage-community-plugin-redhat-argocd",";","Disabled"
 "Azure ","@backstage/plugin-scaffolder-backend-module-azure","Backend","0.1.14","Community Support","./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-azure-dynamic",";","Disabled"
 "Azure Devops ","@backstage/plugin-azure-devops","Frontend","0.4.4","Community Support","./dynamic-plugins/dist/backstage-plugin-azure-devops",";","Disabled"
 "Azure Devops ","@backstage/plugin-azure-devops-backend","Backend","0.6.5","Community Support","./dynamic-plugins/dist/backstage-plugin-azure-devops-backend-dynamic","`AZURE_TOKEN`;`AZURE_ORG`;","Disabled"
 "Azure Repositories ","@parfuemerie/douglas-scaffolder-backend-module-azure-repositories","Backend","0.3.0","Community Support","./dynamic-plugins/dist/parfuemerie-douglas-scaffolder-backend-module-azure-repositories",";","Disabled"
+"Backend ","@pagerduty/backstage-plugin-backend","Backend","0.6.1","Community Support","./dynamic-plugins/dist/pagerduty-backstage-plugin-backend-dynamic","`PAGERDUTY_API_BASE`;`PAGERDUTY_CLIENT_ID`;`PAGERDUTY_CLIENT_SECRET`;`PAGERDUTY_SUBDOMAIN`;","Disabled"
 "Bitbucket Cloud ","@backstage/plugin-catalog-backend-module-bitbucket-cloud","Backend","0.2.9","Community Support","./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-cloud-dynamic","`BITBUCKET_WORKSPACE`;","Disabled"
 "Bitbucket Cloud ","@backstage/plugin-scaffolder-backend-module-bitbucket-cloud","Backend","0.1.12","Community Support","./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-cloud-dynamic",";","Disabled"
 "Bitbucket Server ","@backstage/plugin-catalog-backend-module-bitbucket-server","Backend","0.1.36","Community Support","./dynamic-plugins/dist/backstage-plugin-catalog-backend-module-bitbucket-server-dynamic","`BITBUCKET_HOST`;","Disabled"
 "Bitbucket Server ","@backstage/plugin-scaffolder-backend-module-bitbucket-server","Backend","0.1.12","Community Support","./dynamic-plugins/dist/backstage-plugin-scaffolder-backend-module-bitbucket-server-dynamic",";","Disabled"
-"Bulk Import ","@janus-idp/backstage-plugin-bulk-import-backend","Backend","1.3.0","Community Support","./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic",";","Disabled"
+"Bulk Import ","@janus-idp/backstage-plugin-bulk-import-backend","Backend","1.5.3","Community Support","./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic",";","Disabled"
```

Comment:
```suggestion
"Bulk Import ","@janus-idp/backstage-plugin-bulk-import-backend","Backend","1.5.3","Red Hat Tech Preview","./dynamic-plugins/dist/janus-idp-backstage-plugin-bulk-import-backend-dynamic",";","Disabled"
```

🤔 Why is the Bulk Import Backend plugin still listed as `Community Support` here? In 1.5.3, it is supposed to be marked as TP [here](https://github.com/janus-idp/backstage-plugins/blob/%40janus-idp/backstage-plugin-bulk-import-backend%401.5.3/plugins/bulk-import-backend/package.json). Did I miss something?

Reply:
Here's the code that converts plugin metadata into dynamic plugins table data - see https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/blob/main/modules/dynamic-plugins/rhdh-supported-plugins.sh#L173-L186:

```
        # default to community unless it's a RH-authored plugin
        Support_Level="Community Support"
        keywords=$(jq -r '.keywords' "$j")
        author=$(jq -r '.author' "$j")
        if [[ $author == "Red Hat" ]]; then 
            if [[ $keywords == *"support:production"* ]]; then
                Support_Level="Production"
            elif [[ $keywords == *"support:tech-preview"* ]]; then
                # mark Tech Preview wrappers as Community Supported
                if [[ "$j" != *"/wrappers/"* ]]; then 
                    Support_Level="Red Hat Tech Preview"
                fi
            fi
        fi
```

Reply:
Whoops, I missed that `author` field 🤦🏿
Okay, as discussed, it'll be fixed in 1.4.
Thanks.